cmd/cainjector/app/start.go

```diff
@@ -120,47 +108,66 @@ func (o InjectorControllerOptions) runCertificateBasedInjector(stopCh <-chan str
 		LeaderElectionID:        "cert-manager-cainjector-leader-election",
 		MetricsBindAddress:      "0",
 	})
-
 	if err != nil {
-		o.log.Error(err, "error creating manager")
-		os.Exit(1)
+		return fmt.Errorf("error creating manager: %v", err)
```

Comment:
```suggestion
		return fmt.Errorf("error creating manager: %w", err)
```

Reply:
I know what you mean and I went through a phase of using `%w` everywhere, but I've stopped doing it unless I particularly want to to be able to unwrap the underlying error....otherwise its causing an unnecessarily leaky abstraction.